common/libs/values.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type Big from 'big.js';
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
I'm all for this. I also think we might want to replace all `Big`s with `Bignumber`s. It seems to have a lot of this baked in, plus it's what v3 used https://github.com/MikeMcl/bignumber.js/